lib/chef_zero/endpoints/actor_endpoint.rb

```diff
@@ -8,6 +8,16 @@ module Endpoints
     # /organizations/ORG/users/NAME
     # /users/NAME
     class ActorEndpoint < RestObjectEndpoint
+
+      def get(request)
+        result = super
+        user_data = FFI_Yajl::Parser.parse(result[2], :create_additions => false)
```

Comment:
FWIW RestBase [defines `parse_json` and `to_json` helpers](https://github.com/chef/chef-zero/blob/75e1d346dedbc40cd325c1f0dee2275bcace9b6a/lib/chef_zero/rest_base.rb#L286-L292). (I discovered this too late for my last PR).